src/tools/clippy/clippy_lints/src/lib.rs

```diff
@@ -131,7 +131,6 @@ mod extra_unused_type_parameters;
 mod fallible_impl_from;
 mod float_literal;
 mod floating_point_arithmetic;
-mod fn_null_check;
```

Comment:
you also need to delete `fn_null_check.rs`

Reply:
whoops, fixed!